plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java

```diff
@@ -98,21 +98,21 @@ public void testCacheFileOperations()
                         .add(new CacheOperation("Alluxio.readCached", "00000000000000000002.json", 0, 636))
```

Comment:
why all these below went up?

Reply:
started populating 2 new fields is_exact_min/max in the parquet footer